pf/tfbridge/provider_checkconfig.go

```diff
@@ -83,9 +83,13 @@ func (p *provider) CheckConfigWithContext(
 	// Store for use in subsequent ApplyDefaultInfoValues.
 	p.lastKnownProviderConfig = news
 
-	checkFailures, err := p.validateProviderConfig(ctx, urn, news)
-	if err != nil {
-		return nil, nil, err
+	var checkFailures []plugin.CheckFailure
+	if !news.ContainsUnknowns() {
+		var err error
+		checkFailures, err = p.validateProviderConfig(ctx, urn, news)
```

Comment:
I really wish we were able to take a more precise action here. This gives up validation on required as well as types.

I'll let you know if I think of something else.

Reply:
Does TF run validators on unknown values?

Reply:
We still need to fix the bug, this would only be a workaround if we cannot fix the bug fast enough. It seems yes TF runs validators on objects in presence of unknowns, because the validators have clauses testing if a given tftypes.Value is unknown or not; I'll give it a few hours tomorrow to try to see why we are passing an invalid-typed value there.